src/plugins/github/fetchGitHubRepo.js

```diff
@@ -42,111 +45,100 @@ export default function fetchGitHubRepo(
-                  id
-                  body
-                  author {
-                    ...whoami
-                  }
-                }
-              }
-            }
-          }
-        }
-      }
-    }
-  }
-
-  fragment whoami on Actor {
-    __typename
-    login
-    ... on User {
-      id
-    }
-    ... on Organization {
-      id
-    }
-    ... on Bot {
-      id
-    }
-  }
-  `;
+  const query = GraphQLQueries.stringify.body(
+    createQuery(),
+    GraphQLQueries.inlineLayout()
+  );
   const variables = {repoOwner, repoName};
   const payload = {query, variables};
   return postQuery(payload, token);
 }
 
 const GITHUB_GRAPHQL_SERVER = "https://api.github.com/graphql";
 
+function createQuery(): Body {
+  const b = GraphQLQueries.build;
+  const makePageInfo = () => b.field("pageInfo", {}, [b.field("hasNextPage")]);
+  const makeAuthor = () => b.field("author", {}, [b.fragmentSpread("whoami")]);
+  const body: Body = [
+    b.query(
+      "FetchData",
+      [b.param("repoOwner", "String!"), b.param("repoName", "String!")],
+      [
+        b.field(
+          "repository",
+          {owner: b.variable("repoOwner"), name: b.variable("repoName")},
+          [
+            b.field("issues", {first: b.literal(100)}, [
+              makePageInfo(),
+              b.field("nodes", {}, [
+                b.field("id"),
+                b.field("title"),
+                b.field("body"),
+                b.field("number"),
+                makeAuthor(),
+                b.field("comments", {first: b.literal(20)}, [
+                  makePageInfo(),
+                  b.field("nodes", {}, [
+                    b.field("id"),
+                    makeAuthor(),
+                    b.field("body"),
+                    b.field("url"),
+                  ]),
+                ]),
+              ]),
+            ]),
+            b.field("pullRequests", {first: b.literal(100)}, [
+              makePageInfo(),
+              b.field("nodes", {}, [
+                b.field("id"),
+                b.field("title"),
+                b.field("body"),
+                b.field("number"),
+                makeAuthor(),
+                b.field("comments", {first: b.literal(20)}, [
+                  makePageInfo(),
+                  b.field("nodes", {}, [
+                    b.field("id"),
+                    makeAuthor(),
+                    b.field("body"),
+                    b.field("url"),
+                  ]),
+                ]),
+                b.field("reviews", {first: b.literal(10)}, [
+                  makePageInfo(),
+                  b.field("nodes", {}, [
+                    b.field("id"),
+                    b.field("body"),
+                    makeAuthor(),
+                    b.field("state"),
+                    b.field("comments", {first: b.literal(10)}, [
+                      makePageInfo(),
+                      b.field("nodes", {}, [
+                        b.field("id"),
```

Comment:
Please get the comment url too

Reply:
Will do. (The query was correct when written; more stuff has been merged onto master since then. The tests will catch this when I rebase, and then I'll fix it.)